flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/SourceOperator.java

```diff
@@ -333,6 +333,7 @@ public void open() throws Exception {
         // restore the state if necessary.
         final List<SplitT> splits = CollectionUtil.iterableToList(readerState.get());
         if (!splits.isEmpty()) {
+            LOG.info("Restoring split(s) state to reader {}.", splits);
```

Comment:
Because `sourceReader.addSplits(splits)` in the next line will log the details of the splits, maybe we can simply have `LOG.info("Restoring state for {} split(s) to reader.", splits.size())`?

Reply:
> Because `sourceReader.addSplits(splits)` in the next line will log the details of the splits, maybe we can simply have `LOG.info("Restoring state for {} split(s) to reader.", splits.size())`?

yeah. i think u are right. because next SourceReaderBase will print adding splits details.

Reply:
@becketqin Hi, qin, i updated this pr, PTAL. thanks a lot.